app/controllers/admin/merchants_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::MerchantsController < ApplicationController
+
+  def show
+    @merchant = Merchant.find_by_id(params[:merchant_id])
+  end
```

Comment:
I also created a Admin:MerchantsController but am only using index and update actions as of now, so we should be good.